### ISSUE TITLE
Missing upload caption file button on Android

### ISSUE THREAD

**Issue by andreaxhearts** [bluesky-social/social-app#11652]
<img width="360" height="500" alt="screenshot of Bluesky Android app, creating new post, that has a video preview with black video. the alt text field is opened. text: 'cc'. (inside a square). 'Alt text'. 'Add alt text (optional)'" src="https://github.com/user-attachments/assets/e5709215-f0c5-49f2-b503-409e1e7c9907" />

### Steps to Reproduce

1. Compose new post. (pencil icon bottom-left)
2. Upload media > ⋮ menu > Browse…
3. Select a video file in file picker.
4. Tap the text that says "CC Alt text" below the video preview
5. Only a text field for Alt text is shown.

### What platform(s) does this occur on?

Android

### Device Info

OnePlus 7 Pro, Android 12

### What version of the app are you using?

1.131.1

### Additional Information

There does not appear to be an option to upload a .vtt caption file on the Android. 
The graphic for CC is visible near the Alt text option, suggesting subtitles are supported. I'm not able to provide captions with my .mp4 file.

**Comment by mozzius** [comment]
This is only supported on web for the time being
